qa/tasks/cephfs/test_volumes.py

```diff
@@ -607,6 +613,42 @@ def test_volume_rename_for_more_than_one_data_pool(self):
         # data pool names unchanged
         self.assertCountEqual(orig_data_pool_names, list(self.fs.data_pools.values()))
 
+    def test_volume_info(self):
+        """Tests the 'fs volume info' command
+        """
+        subvol_md = ["volname", "gid", "size", "subvolumegroups", "pending_subvolume_deletions", "mon_addrs"]
+
+        newvolname = self._generate_random_volume_name()
+        group = self._generate_random_group_name(count=2)
+        
+        # create volume        
+        self._fs_cmd("volume", "create", self.volname)        
+        self._fs_cmd("subvolumegroup", "create", self.volname, group[0])
+
+        # get volume metadata
+        vol_info = json.loads(self._get_volume_info(self.volname))
+        for md in subvol_md:
+            self.assertIn(md, vol_info, f"'{md}' key not present in metadata of volume")
+
+        self.assertEqual(vol_info["subvolumegroups"], 1, "Number of subvolumegroups should be one. ")
+        self.assertEqual(vol_info["size"], 0, "Size should be zero when volumes directory is empty. ")
+        self.assertEqual(vol_info["volname"], self.volname, f"Name of volume should be {self.volname}")        
+        
+        # create group
```

Comment:
Please add separate tests. It's good to cover different test scenarios separately.